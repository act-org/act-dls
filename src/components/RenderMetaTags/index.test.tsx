/**
 * Copyright (c) ACT, Inc. and its affiliates.
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * @prettier
 */

import * as React from 'react';
import renderer from 'react-test-renderer';

import { RenderMetaTags } from '.';

describe('RenderMetaTags', () => {
  test.each(THEMES)('%s theme matches the snapshot', theme => {
    const tree = renderer
      .create(
        <RenderMetaTags
          description="Some description."
          imageUrl="https://www.act.org/etc/designs/act/clientlibs/act-base/img/logo-act-blue-300.png"
          keywords={['keyword1', 'keyword2', 'keyword3']}
          title="Some Title"
        />,
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
