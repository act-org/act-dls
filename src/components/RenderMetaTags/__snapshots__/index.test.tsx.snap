// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RenderMetaTags matches the snapshot 1`] = `
Array [
  <meta
    content="Some description."
    name="description"
  />,
  <meta
    content="Some description."
    property="og:description"
  />,
  <meta
    content="Some description."
    name="twitter:text:description"
  />,
  <meta
    content="https://mawi.et-act.org"
    property="og:image"
  />,
  <meta
    content="https://mawi.et-act.org"
    name="twitter:image"
  />,
  <meta
    content="keyword1,keyword2,keyword3"
    name="keywords"
  />,
  <title
    itemProp="name"
    lang="en"
  >
    Some Title
  </title>,
  <meta
    content="Some Title"
    property="og:title"
  />,
  <meta
    content="Some Title"
    name="twitter:title"
  />,
]
`;
